azuredevops/internal/service/core/data_team.go

```diff
@@ -61,7 +61,15 @@ func dataTeamRead(d *schema.ResourceData, m interface{}) error {
 	projectID := d.Get("project_id").(string)
 	teamName := d.Get("name").(string)
 
-	team, members, administrators, err := readTeamByName(d, clients, projectID, teamName)
+	var top int
+	data, ok := d.GetOk("top")
```

Comment:
`top` should be defined in the schema like other property.
Ref: https://github.com/microsoft/terraform-provider-azuredevops/blob/main/azuredevops/internal/service/core/data_team.go#L16